### DISCUSSION TITLE
Correct location of codelists

### DISCUSSION THREAD

**Discussion by techteam-bln** [ldproxy/ldproxy#1100] [General]
Hallo,
ich versuche grade mich in euer Programm einzuarbeiten und das Demo auf Basis der Daten in eurem [Demo Config Repo](https://github.com/ldproxy/demo) nachzubauen. Dies funktioniert auch recht gutm ebenso auch eigene Test-Services.
Was nicht klappt ist das Beispiel [3D Buildings in Cologne (LoD2)](https://demo.ldproxy.net/cologne_lod2), hier findet  ldproxy nicht die bereitgestellen Codelists.
Ich habe unterhalb des als /ldproxy/data/ eingebunden Verzeichnisses mehere Pfade ausprobiert, leider ohne Erfolg.
Beim generieren der Informationen kommt innerhalb der Console immer wieder die Warnung: 
Skipping CODELIST transformation for property 'function', codelist <NAME_DER_CODE_LIST> not found.

Danke im voraus
Heiko

#####################################
Hello,
I'm currently trying to familiarize myself with your program and recreate the demo based on the data in your [Demo Config Repo](https://github.com/ldproxy/demo). This also works quite well as well as our own test services.
What doesn't work is the example [3D Buildings in Cologne (LoD2)](https://demo.ldproxy.net/cologne_lod2), here ldproxy cannot find the code lists provided.
I tried several paths below the directory included as /ldproxy/data/, unfortunately without success.
When generating the information, the warning keeps coming up in the console:
Skipping CODELIST transformation for property 'function', codelist <NAME_DER_CODE_LIST> not found.

**Comment by cportele** [comment]
Hallo @techteam-bln, die aktuelle Konfiguration hat die Codelisten in ["values/codelists"](https://github.com/ldproxy/demo/tree/main/values/codelists). Das wird erst seit ca. 2 Wochen im `ldproxy:next`-Image unterstützt (Änderung im Rahmen von PR #1087 und Issue #1077). Ein älteres `ldproxy:next`-Image würde die Codelisten in ["store/entities/codelists"](https://docs.ldproxy.net/de/application/40-store.html) bzw. ["entities/instances/codelists"](https://docs.ldproxy.net/de/application/41-store-new.html) erwarten (die Dokumentation enthält noch nicht die letzte Änderung).

**Reply by techteam-bln** [reply]
Danke für die schnelle Antwort, jetzt funktioniert alles wie gewollt.
Schönes Wochenende. :)

**Comment by techteam-bln** [comment]
resolved
